test/gurka/test_tagged_values.cc

```diff
@@ -31,6 +31,7 @@ class TaggedValues : public ::testing::Test {
                           H
     )";
 
+    // TODO: does the graphparser make sure we dont ever try to set layer to 0 since its the default
```

Comment:
can you elaborate?

Reply:
I think kevin added the comment here. the gist is the parser currently suppose kLayer doesn't include any null bytes. if in any case kLayer is set to zero the parser will return an problematic empty string. So we'd better make sure kLayer is never set to 0, or use special parser for it which can parse null byte as the one for kLandmark.